hack/dind-run-operators.sh

```diff
@@ -1,8 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
It is useful for me to have a bash script rather than doing a copy and paste from the tutorial.

Reply:
this script is out of date, it will include `./manifests/local-dind/setup.sh` -> `./manifests/local-dind/pv-hosts.sh` -> `./manifests/local-dind/pv-create.sh`.

but the pv create is replaced by `manifests/local-dind/dind-cluster-v1.12.sh`.

and many users use `pv-create.sh` in their production setup, this is dangerous.

So we may better use [this document](https://github.com/pingcap/tidb-operator/blob/master/docs/local-dind-tutorial.md) for local dind setup instead.

Reply:
Yes, the script is out of date. Can we update it to use the commands in that document?

```
manifests/local-dind/dind-cluster-v1.12.sh up
kubectl apply -f manifests/crd.yaml
helm install charts/tidb-operator --name=tidb-operator --namespace=tidb-admin
helm install charts/tidb-cluster --name=demo --namespace=tidb
```

Reply:
of course